source/NPC.cpp

```diff
@@ -753,14 +754,15 @@ void NPC::DoActions(const ShipEvent &event, bool newEvent, PlayerInfo &player, U
 	// handle the first such event, because a ship can't actually be
 	// destroyed multiple times.
 	if(type == ShipEvent::DESTROY && !newEvent)
```

Comment:
Presumably? 

```suggestion
	if((type & ShipEvent::DESTROY) && !newEvent)
```

Reply:
I wouldn't expect the second DESTROY event that gets sent to occur at the same time as others, but probably a good idea.